src/main/java/study/domain/Calculator.java

```diff
@@ -0,0 +1,42 @@
+package study.domain;
+
+import study.domain.util.FormulaParser;
+
+import java.util.regex.Pattern;
+
+public class Calculator {
+    private static final Pattern REG_EXP = Pattern.compile("^\\d+$");
+    private final FormulaParser formulaParser;
```

Comment:
```suggestion
    private static final Pattern REG_EXP = Pattern.compile("^\\d+$");
    
    private final FormulaParser formulaParser;
```

클래스 변수와 인스턴스 변수 사이에 공백이 있다면 가독성이 올라갈 것 같아요 😄

Reply:
omg .... intellij 어느 기능으로 인해 useage 라고 나오는지 ... 
![image](https://user-images.githubusercontent.com/55230332/233825439-c55f8391-b673-44c1-b12b-b57ae733ec96.png)

마치 떨어져 있는것 같은 느낌이 들었지만 사실상 붙어있었네요 ㅠㅠㅠ